.gitmodules

```
@@ -1,3 +1,3 @@
 [submodule "scripts/firmware"]
 	path = scripts/firmware
-	url = git://git.studentrobotics.org/boards/firmware.git
+	url = https://github.com/srobo-legacy/boards-firmware.git
```

Transferred into main org. https://github.com/srobo/boards-firmware
```suggestion
	url = https://github.com/srobo/boards-firmware.git
```